ember-cli-qunit "^1.2.0" not found when attempting to upgrade to ember-cli v2.2.0-beta.1

Just attempted to upgrade to ember-cli v2.2.0-beta.1 and hit a snag when installing ember-cli-quint `1.2.0` which was listed in the change logs as a required change.

I see ember-cli-qunit `1.2.0` is tagged as a release here:

https://github.com/ember-cli/ember-cli-qunit/releases

But it doesn't seem to be in NPM which lists `1.1.0` as the latest release:

https://www.npmjs.com/package/ember-cli-qunit


```
npm ERR! Linux 3.10.0-229.el7.x86_64
npm ERR! argv "/home/work/.nvm/versions/node/v4.2.2/bin/node" "/home/work/.nvm/versions/node/v4.2.2/bin/npm" "install"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.7
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: ember-cli-qunit@'>=1.2.0 <2.0.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.1","0.0.2","0.0.3","0.0.4","0.0.5","0.0.6","0.0.7","0.1.0","0.1.1","0.1.2","0.2.0","0.3.0","0.3.1","0.3.2","0.3.3","0.3.4","0.3.5","0.3.6","0.3.7","0.3.8","0.3.9","0.3.10","0.3.11","0.3.12","0.3.13","0.3.14","0.3.15","0.3.16","0.3.17","0.3.18","0.3.19","0.3.20","0.4.0","1.0.0","1.0.1","1.0.2","1.0.3","1.0.4","1.0.5","1.1.0"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'test-project'
npm ERR! notarget

npm ERR! Please include the following file with any support request:
npm ERR!     /home/work/workspace/test-project/npm-debug.log
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,7 +11,7 @@ The following changes are required if you are upgrading from the previous versio
     * `ember-ajax` to `0.7.1`
     * `ember-cli-dependency-checker` to `^1.2.0`
     * `ember-cli-sri` to `^1.2.0`
-    * `ember-cli-qunit` to `^1.2.0`
+    * `ember-cli-qunit` to `^1.1.0`
     * `ember-data` to `^2.2.1`
   + Update the following entries in your `bower.json`:
     * `ember` to `2.2.0`

